packages/rum-core/test/common/fetch-patch.spec.js

```diff
@@ -56,8 +56,10 @@ describe('xhrPatch', function() {
       expect(events.map(e => e.event)).toEqual(['schedule'])
       promise.then(function(resp) {
         expect(resp).toBeDefined()
-        expect(events.map(e => e.event)).toEqual(['schedule', 'invoke'])
-        done()
+        Promise.resolve().then(function() {
```

Comment:
Do we need this still even after doing Promise.resolve instead of setTimeout?

Reply:
Yes, we actually need this because of the Promise.resolve in the fetch patch, since at this point the invoke task is not called yet.